### PR TITLE
Confirm nothing breaks after rules_docker incompatible change

### PR DESCRIPTION
**Pull request by alex1545**
rules_docker address this [incompatible Bazel change](https://github.com/bazelbuild/bazel/issues/7347) in this [PR](https://github.com/bazelbuild/rules_docker/pull/755).
This PR is the verify we don't break things in this project.

### CODE REVIEW

**Comment by alex1545** [comment]
This PR will need to be merged after our rules_docker [PR](https://github.com/bazelbuild/rules_docker/pull/755) is merged.

File: WORKSPACE [hunk 1]
```diff
@@ -39,19 +46,10 @@ register_toolchains(
     "@io_bazel_rules_docker//toolchains/docker:default_osx_toolchain",
 )
 
-load(
-    "@io_bazel_rules_docker//docker:docker.bzl",
-    "docker_pull",
-    "docker_repositories",
-)
+load("@io_bazel_rules_docker//docker:docker.bzl" "docker_pull")
```
Comment (sharifelgamal) [inline WORKSPACE]: this is missing a comma

Comment (alex1545) [reply WORKSPACE]: Oops

**Comment by sharifelgamal** [comment]
The current test error was fixed by something that was merged to master after this PR was created. Can you rebase so I can rerun the tests?

**Comment by alex1545** [comment]
Can you please try again?

**Comment by xingao267** [comment]
This is no longer needed after https://github.com/GoogleContainerTools/base-images-docker/pull/314

**Pull request closed without merging**